install/ecs/main.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
```

Comment:
can't see terraform backend for keeping state

Reply:
This deployment example, will add comment